test/fixtures/privileges.sql

```diff
@@ -62,9 +62,12 @@ GRANT USAGE ON SEQUENCE
 TO postgrest_test_anonymous;
 
 -- Privileges for non anonymous users
+GRANT USAGE ON SCHEMA test TO postgrest_test_default_role;
```

Comment:
This [test](https://github.com/begriffs/postgrest/blob/master/test/Feature/AuthSpec.hs#L128) failed, it responded with ``` {"hint":null,"details":null,"code":"42501","message":"permission denied for schema test"}```.

Seems that when switching roles the ```postgrest_test_default_role``` had the privilege of only using the ```public``` schema and not the api exposed ```test``` schema.